instructure/canvas-ios
# Student View

refs: MBL-13855
affects: teacher
release note: Adds Student View

Test plan:
* Install both Teacher and Student builds from this PR
* Log in as a teacher (narmstrong/t)
* Tap in to a course and choose Student View from course nav (CS 1400 > Student View)
* Student app should launch and you should be logged in as the Test Student
* Poke around the course and do student things (submit an assignment, reply to a discussion, etc)
* Tap the masquerading avatar and make sure it prompts you to "Leave Student View"
* Tap Cancel to make sure it works
* Tap OK and see that you are logged out
* Delete the Student App
* In Teacher app, tap to launch Student View again
* See that you are redirected to the App Store page for Student app
